### PR TITLE
add timeout param to default requester configs

### PR DESCRIPTION
**Pull request by poocart**
Per title. By default `axios` has no timeout set and if it happens then it is not rejected as exception, but is handled as resolved promise as `200` response with timeout error message.

### CODE REVIEW

File: src/tests/unit/configuration.spec.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 */
+import nock = require('nock');
```
Comment (jsidorenko) [inline src/tests/unit/configuration.spec.ts]: import + required? o_O

File: src/utils/requester-configurations/put.ts [hunk 2]
```diff
@@ -27,6 +27,7 @@ export default {
   headers: {},
   data: {},
   json: true,
+  timeout: 10000,
```
Comment (jsidorenko) [inline src/utils/requester-configurations/put.ts]: How about extracting this timeout value into some constant so we could re-use it across the app?

File: src/tests/unit/configuration.spec.ts (hunk 1 above)
Comment (poocart) [reply src/tests/unit/configuration.spec.ts]: lol, good one, sry :D IDE did this and my fault for not paying attention to this!

File: src/utils/requester-configurations/put.ts (hunk 2 above)
Comment (poocart) [reply src/utils/requester-configurations/put.ts]: Makes sense, I would also like to add different value if test is running and pass it to unit tests 👍

File: src/tests/unit/configuration.spec.ts (hunk 1 above)
Comment (poocart) [reply src/tests/unit/configuration.spec.ts]: double sorry – it's not IDE, it's actually throughout the code, will change everywhere ¯\_(ツ)_/¯

**Pull request merged**